### PR TITLE
Fixed issue where initCheck : false was not working #22

### PR DESCRIPTION
**Pull request by DhyeyMoliya**
Fixes Issue : https://github.com/chainlist/svelte-forms/issues/22

Description : Used `field.dirty` and `config.initCheck` to control css class `add/remove`

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -141,6 +142,8 @@ export function form(fn, config = {}) {
     config
   );
 
+  storeValue.update(value => ({ ...value, initCheck: config.initCheck }));
```
Comment (AnthyG) [inline src/index.js:145]: Am I missing something or could this not just be added to \`storeValue\` right away :thinking: (so move `config` above `const storeValue = ...` (L128)

File: src/index.js [hunk 2]
```diff
@@ -95,13 +95,14 @@ export function bindClass(
...
     }
     const field = context.fields[key];
-
-    if (field.valid) {
-      node.classList.add(valid);
-      node.classList.remove(invalid);
-    } else {
-      node.classList.remove(valid);
-      node.classList.add(invalid);
+    if ((!context.initCheck && field.dirty) || context.initCheck) {
```
Comment (AnthyG) [inline src/index.js:98]: :+1:

**Review by AnthyG — COMMENTED** [review]
LGTM except for the part of updating `storeValue` right after defining it (maybe I don't fully understand that part tho, reactivity and such :D)

File: src/index.js (hunk 1 above)
Comment (DhyeyMoliya) [reply src/index.js:145]: Can be done @AnthyG , I did not want to move the structure much, that's why I haven't moved it.

**Comment by DhyeyMoliya** [comment]
@AnthyG  Should I make your suggested changes and push again?

**Comment by AnthyG** [comment]
Yeah, if there hasn't been any specific reason why you used `storeValue.update` instead of moving the config infront of `storeValue` declaration, do it :D

**Comment by DhyeyMoliya** [comment]
@AnthyG and @chainlist I have updated the code fixing the issue(https://github.com/chainlist/svelte-forms/issues/22) for `initCheck : false`. I hope this gets merged soon.

**Review by AnthyG — APPROVED** [review]
:+1:

**Comment by chainlist** [comment]
@DhyeyMoliya thanks for the changes!

**Pull request merged**